src/entity2/user.ts

```diff
@@ -0,0 +1,49 @@
+import { Entity, PrimaryGeneratedColumn, Column, BaseEntity } from 'typeorm';
+
+@Entity()
+export class User extends BaseEntity {
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
+
+    @Column({ unique: true })
+    username: string;
```

Comment:
I wouldn't bother having a username separate from an eMail address.

Reply:
I will drop usename in the next commit to this PR.